js/atree_constants.js

```diff
@@ -4759,7 +4759,7 @@ const atrees = {
                             "name": "eConvBase:3.Lightning Damage"
                         }
                     ],
```

Comment:
these have to be split up into two -- the old (current) code is applying a 15% scaling to both `nConvBase` and `eConvBase`, i believe in the changelog only the earth scaling was reduced